contracts/src/ethereum/TokenBridge.sol

```diff
@@ -170,11 +187,10 @@ contract TokenBridge is
         messagingContract.consumeMessageFromL2(l2TokenBridge, payload);
     }
 
-    function receiveRewards(address l1Token, address recipient, uint256 amount) isApprovedToken(l1Token) external {
-        consumeBridgeRewardMessage(l1Token, recipient, amount);
+    function receiveRewards(IStaticATokenLM staticAToken, address recipient, uint256 amount) isApprovedToken(address(staticAToken)) external {
```

Comment:
why passing tokens instead of their address?

Reply:
same question

Reply:
The token and the address are the same thing. They're just type aliases with call information for the compiler.